ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/deprovision/cucushift-installer-rehearse-aws-ipi-deprovision-chain.yaml

```diff
@@ -1,6 +1,7 @@
 chain:
   as: cucushift-installer-rehearse-aws-ipi-deprovision
   steps:
+  - ref: junit2reportportal
```

Comment:
Will the depredation steps be reported if putting `junit2reportportal` to the end? I prefer to report all deprovision steps' results, @jianlinliu WDYT?

Reply:
Yes, it can report all steps that executed before this step. I'll update